frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'common/types/types';
+import { useState } from 'react';
+
+import styles from '../../uam/components/group-creation/group-creation.module.scss';
+
+type Props = {
+  styles: {
+    wrapperClass: string;
+    inputClass: string;
+  };
+};
+
+const Checkbox: FC<Props> = ({ styles }) => {
+  const [isChecked, setIsChecked] = useState(false);
```

Comment:
```suggestion
```
you need to use `useForm` api
All form components should work only with this hook

Reply:
I have read useForm docs, and as far as I understood we are free to use useState inside of a checkbox. (because otherwise I had some issues, but maybe I was doing something wrong)